db/migrate/20220204114732_drop_proceeding_type_tables.rb

```diff
@@ -0,0 +1,13 @@
+class DropProceedingTypeTables < ActiveRecord::Migration[6.1]
+  def up
+    drop_table :application_proceeding_types, force: :cascade
+    drop_table :proceeding_types, force: :cascade
+    drop_table :scope_limitations, force: :cascade
+    drop_table :proceeding_type_scope_limitations, force: :cascade
+    drop_table :application_proceeding_types_scope_limitations, force: :cascade
```

Comment:
Possibly some other tables that could be dropped here too:

`application_proceeding_types_linked_children`
`default_cost_limitations`
`service_levels`

Reply:
dropped them